docs/how-to/custom-env.txt

```diff
@@ -113,24 +112,24 @@ This can be configured in  your experiment configuration like below:
 Custom Docker Images in Determined
 ----------------------------------
 
-While our official images contain all the dependencies you need for basic
+While our official images contain all the dependencies needed for basic
 workloads, many workloads have extra dependencies. If those extra dependencies
-are quick to install, you may want to consider using the ``startup-hook.sh``.
-However, some dependencies take too long to install via the
+are quick to install, you may want to consider using ``startup-hook.sh``.
+However, some dependencies take too long to install via
 ``startup-hook.sh``. In this situation, we suggest building your own Docker
-image and publishing to a Docker registry like Docker Hub
-(https://hub.docker.com/). Below you will find an example of a Dockerfile. For
-more information on building and publishing Docker images to DockerHub,
+image and publishing to a Docker registry like `Docker Hub
+<https://hub.docker.com/>`__. Below you will find an example of a Dockerfile. For
+more information on building and publishing Docker images to Docker Hub,
 consider following https://docs.docker.com/get-started/.
 
 The example Dockerfile below installs ``conda`` and ``pip`` dependencies based
 on their respective dependency file format.
 
 .. warning::
 
-	It is important to not install the TensorFlow, PyTorch, Horovod, or Apex
-	packages as this will conflict with the base packages that are installed into
-	Determined's official environments.
+   It is important to not install the TensorFlow, PyTorch, Horovod, or Apex
```

Comment:
non-blocking: Worth highlighting this somehow? Bold, italics, something else?

Reply:
It's still in a box with a big red bar above it, so I think I'm going to call it good.